webpack.config.js

```diff
@@ -25,6 +25,7 @@ const config = {
   output: {
     filename: "[name].js",
     path: pathLib.resolve(__dirname, "build"),
+    hashFunction: "xxhash64",
```

Comment:
Weird compatibility / vulnerability issue resolved via the solution presented here: https://stackoverflow.com/a/73027407

Was seeing the error `Error: error:0308010C:digital envelope routines::unsupported`

Reply:
Sure thing